fjord/base/migrations/0003_remove_ditchchart_waffle_flag.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
This needs a module-level docstring explaining what this migration does.

http://fjord.readthedocs.org/en/latest/db.html#creating-a-data-migration

See this as an example:

https://github.com/mozilla/fjord/blob/master/fjord/heartbeat/migrations/0004_auto_20150225_1232.py#L2